internal/request_test.go

```diff
@@ -1,24 +0,0 @@
-package internal
-
-import (
-	"net/http"
-	"testing"
-)
-
-func TestDeviceIDFromRequest(t *testing.T) {
```

Comment:
We can no longer directly determine the device id (i.e. the unique id used by the proxy to identify a device) by the information provided in the request.